Scripts/Monitor CPU Fan.lua

```diff
@@ -0,0 +1,73 @@
+-- ISC License
+--
+-- Copyright © 2017, Simonas Kazlauskas
+-- Copyright © 2018, Pedro Fanha
+--
+-- Permission to use, copy, modify, and/or distribute this software for any
+-- purpose with or without fee is hereby granted, provided that the above1
+-- copyright notice and this permission notice appear in all copies.
+--
+-- THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
+-- REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+-- AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
+-- INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+-- LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE
+-- OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+-- PERFORMANCE OF THIS SOFTWARE.
+
+-- Min and Max speed of the CPU fan
+local fan = {700, 1800}
+
+local color_min = 0
```

Comment:
I feel like the delay is too high - it doesn't look at all smooth. Maybe you can do something about it?

Reply:
This is set depending on the temperature, I don't think I can make it any smoother than decreasing the delay.